build.mjs

```diff
@@ -0,0 +1,9 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { overrideGQLOperations } from '@dropins/build-tools/gql-extend.js';
+
+overrideGQLOperations([
+  {
+    npm: '@dropins/storefront-cart',
+    operations: [],
```

Comment:
I think we should add a minimal example, like in the PR description.

Reply:
@sirugh, same as with Payment Providers, shouldn't that be part of the documentation? Unless there is a default out-of-the-box integration that requires it.

Reply:
You could just include it commented out, but either works :)